File: Python/starsze/czesc.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-
#
#  czesc.py
#  

ROK_TERAZ = 2017
ROK_PYTHON = 1991

def parzyste():
    ile = list(range(0, n+1, 2))
    print(ile)
    return len(ile)

def main(args):
    imie = input("Jak się nazywasz? ")
    print ("Cześć", imie)
    wiek = int(input("Ile masz lat "))
    rok_urodzenia = ROK_TERAZ - wiek
    print ("Urodziłaś się w ", 2017 - wiek)
    
    if rok_urodzenia < ROK_PYTHON:
        print("Jestem starszy")
    elif rok_urodzenia > ROK_PYTHON:
        print("Jestem młodszy")
    else:
        print("Jesteśmy tak samo starzy")
        
    n = int(input("Podaj liczbę: "))
    print("Parzystych:", parzyste(n))
    
    return 0

if __name__ == '__main__':
    import sys
    sys.exit(main(sys.argv))

File: cpp/starsze/czesc.cpp
/*
 * czesc.cpp
 */


#include <iostream>


using namespace std;

#define ROK_TERAZ 2017
#define ROK_CPP 1984

int parzyste (int n) {
    int i;
    for (i=0; i<=n; i+=2)
        cout << i << " ";
    return i/2;
    }

int main(int argc, char **argv)
{
    string a;
    int b;
    int rok_urodzenia;
    int n;
    
    cout << "Cześć. Jak się nazywasz " << endl;
    cin >> a;
    cout << "Cześć " << a << endl;
    cout << "Ile masz lat? " << endl;
    cin >> b;
    rok_urodzenia = ROK_TERAZ - b;
    cout << "Urodziłeś/Urodziłaś się w " << rok_urodzenia << endl;

    if (rok_urodzenia > ROK_CPP) {
        cout << "Jestem starszy" << endl; }
        
    if (rok_urodzenia < ROK_CPP) {
        cout << "Jestem młodszy" << endl; }
        
    if (rok_urodzenia == ROK_CPP) {
        cout << "Jesteśmy tak samo starzy" << endl; }
        
    cout << "Podaj liczbę: " << endl;
    cin >> n;
    cout << "Parzystych: " << parzyste(n) << endl;
    
	return 0;
}

